sdformat_mjcf/src/sdformat_mjcf/mjcf_to_sdformat/converters/world.py

```diff
@@ -27,6 +27,75 @@
 import sdformat as sdf
 
 
+def _is_floating_body(body):
+    freejoint = body.get_children("freejoint")
+    if freejoint:
+        return True
+    all_child_joints = body.get_children("joint")
+    for joint in all_child_joints:
+        if joint.type == "free":
+            return True
+    return False
+
+
+def _add_body_to_model(body, physics, model, modifiers):
+    try:
+        parent_name = body.parent.name
+    except AttributeError:
+        parent_name = "world"
+
+    link = mjcf_body_to_sdf(body,
+                            physics,
+                            body_parent_name=parent_name,
+                            modifiers=modifiers)
+    for camera in body.camera:
+        sensor = mjcf_camera_sensor_to_sdf(camera)
+        if sensor is not None:
+            link.add_sensor(sensor)
+
+    serial_sdf_joints_created = []
+    serial_link_created = [link]
+    small_mass_matrix = MassMatrix3d()
```

Comment:
can you add a comment here ? why did you choose a radius of 1 ?

Reply:
Added a comment in https://github.com/gazebosim/gz-mujoco/pull/87/commits/e60b1c741dcf8113200e4ac00073635431043514. I just picked the unit sphere since it was the easiest to specify. As long as the inertia is small, it doesn't matter.